src/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title = 'Unknown', author = 'Unknown')
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- Here, I believe you are missing a method to add rentals to a book. Remember that the user should be able to add a rental to a book to know if the book has been rented or not, so there should be a method similar to this one:
```
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

So please make sure to work on this issue.